compiler/rustc_hir_analysis/messages.ftl

```diff
@@ -448,6 +448,11 @@ hir_analysis_rpitit_refined = impl trait in impl method signature does not match
     .note = add `#[allow(refining_impl_trait)]` if it is intended for this to be part of the public API of this crate
     .feedback_note = we are soliciting feedback, see issue #121718 <https://github.com/rust-lang/rust/issues/121718> for more information
 
+hir_analysis_rpitit_refined_lifetimes = impl trait in impl method captures fewer lifetimes than in trait
+    .suggestion = add a `use<..>` bound to capture the same lifetimes that the trait does
```

Comment:
we already have a `use` bound :thinking: maybe "modify the use bound..."

Reply:
whoops